dist/tools/eclipsesym/cmdline2xml.sh

```diff
@@ -26,6 +26,10 @@ if [ $# -ne 0 ]; then
     exit 2
 fi
 
+if [ `uname` == "Darwin" ] || [ `uname` == "FreeBSD" ]; then
+    function readlink () { echo $(cd $(dirname $2); pwd)/$(basename $2); }
```

Comment:
`function name() {…}` is bash'ism and disregarded, because it is not widely compatible. Better is the POSIX variant `name() {…}`.

Also, it shouldn't hurt do use this function in Linux, too, does it? Then you could remove the guard so users of maybe OpenBSD are happy, too.